Gruntfile.js

```diff
@@ -5,7 +5,6 @@ module.exports = function (grunt) {
 
   function getIncludeFiles() {
     var patterns = [
-      'docroot/themes/humsci/!(humsci_basic)/**/*.scss',
```

Comment:
@heatherdesigns I realized I need to keep these rules in here to exclude humsci basic. the root linting task will still look at everything, but the build task cannot because we have special stuff going on in humsci basic

Reply:
I agree with this. It would be cool if, at some point, we could get everything on the same build task. But that's something we can think about another day.